src/components/Footer/Footer.tsx

```diff
@@ -18,11 +18,13 @@ const year = new Date().getFullYear();
 export const Footer = () => {
   const { t } = useTranslation('app');
 
+  const { NODE_ENV, REACT_APP_DEVELOP_BUILD_VERSION: buildVer } = process.env;
+
   return (
     <StyledFooter>
       <Box>
         <StyledText>
-          {version} &#169; {year}
+          {version} {NODE_ENV !== 'production' && buildVer && `(${buildVer})`} &#169; {year}
```

Comment:
maybe it's better to move it to a variable to improve readability?
minor: `buildVer` - I would prefer the full name of the variable - it's not too big in this case:)

Reply:
done